src/network/networkd-link.c

```diff
@@ -2001,6 +2001,7 @@ static int link_up_handler(sd_netlink *rtnl, sd_netlink_message *m, Link *link)
 
 static int link_up(Link *link) {
         _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *req = NULL;
+        uint8_t ipv6ll_mode;
```

Comment:
Not a matter, but I'd like to move the variable definition just after the line `if (link_ipv6_enabled(link)) {`.